hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/api/handlers/BucketEntityHandler.java

```diff
@@ -126,15 +125,17 @@ public DUResponse getDuResponse(
   public QuotaUsageResponse getQuotaResponse()
           throws IOException {
     QuotaUsageResponse quotaUsageResponse = new QuotaUsageResponse();
-    String[] names = getNames();
-    String bucketKey = getOmMetadataManager().getBucketKey(names[0], names[1]);
-    OmBucketInfo bucketInfo = getOmMetadataManager()
-            .getBucketTable().getSkipCache(bucketKey);
-    long bucketObjectId = bucketInfo.getObjectID();
-    long quotaInBytes = bucketInfo.getQuotaInBytes();
-    long quotaUsedInBytes = getTotalSize(bucketObjectId);
-    quotaUsageResponse.setQuota(quotaInBytes);
-    quotaUsageResponse.setQuotaUsed(quotaUsedInBytes);
+    RootEntityHandler rootEntityHandler = new RootEntityHandler(
+        getReconNamespaceSummaryManager(),
+        getOmMetadataManager(),
+        getReconSCM(),
+        "/"
+    );
+    QuotaUsageResponse rootQuotaUsageResponse = 
```

Comment:
in HDFS, -df returns capacity of the whole file system when df towards a directory, which we'll do similar here in Ozone for bucket.